.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,36 @@
+name: Fuzz
+
+on: [push, pull_request]
+
+jobs:
+
+  fuzz:
+    runs-on: ubuntu-20.04
+    env:
+      HFUZZ_BUILD_ARGS: "--features honggfuzz_fuzz"
+    strategy:
+      fail-fast: false
+      matrix:
+        fuzz_target: [deser_net_msg, deserialize_address, deserialize_amount, deserialize_block, deserialize_psbt, deserialize_script, deserialize_transaction, outpoint_string, uint128_fuzz]
```

Comment:
If we can't generate this from the file list, can CI somewhere fail if this misses a fuzz target?

Reply:
yes, check done in 1aefc1ccf3555d707de39ee31238046aab8c4b47